### PR TITLE
Portfolio Finish mobile version PR

### PR DESCRIPTION
**Pull request by diarisdiakite**
In this PR I:
- Added the recent works section with a header and a projects class with a display of grid
- Added 5 different projects with a display flex
- Added the image placeholder to the project class
- Added an _ul_ tag with the class project-built-with
- Added the button see project
- Added a competences section

### CODE REVIEW

**Comment by diarisdiakite** [comment]
In this Milestone, I:
- Fixed placeholder issue for projects
- Fixed text centering issues
- Added buttons interactions

File: index.html [hunk 1]
```diff
@@ -55,6 +54,227 @@ <h2 class="green">I’m a Full Stack developer from Microverse</h2>
...
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
+
+        <div class="project-buttons">
+          <a href="">See project</a>
+        </div>
+        
+      </div>
```
Comment (mahbubul14) [inline index.html:95]: - [ ] Kindly try to match your design to the Figma design. The section has the margins (top, bottom, left, right) defined in the design. The section uses the font and colors (background and font) defined for the template. :+1: 
 
Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign_work | figma_work

File: index.html [hunk 2]
```diff
@@ -55,6 +54,227 @@ <h2 class="green">I’m a Full Stack developer from Microverse</h2>
...
+            Skills
+          </h3>
+        </div>
+        <ul class="list-competences">
+          <li><a href="">Database management</a></li>
+          <li><a href="">version control</a></li>
+          <li><a href="">CLI</a></li>
+          <li><a href="">Web development</a></li>
+          <li><a href="">API design</a></li>
+        </ul>
```
Comment (mahbubul14) [inline index.html:269]: - [ ] Kindly try to match your design to the Figma design. The section has the margins (top, bottom, left, right) defined in the design. The section uses the font and colors (background and font) defined for the template. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign_myself | figma_myself

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @diarisdiakite,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 3]
```diff
@@ -55,6 +54,228 @@ <h2 class="green">I’m a Full Stack developer from Microverse</h2>
...
+          <h3>
+            Languages
+          </h3>
+        </div>
+        <ul class="list-competences">
+          <li><a href="">Javascript</a></li>
+          <li><a href="">Ruby</a></li>
+          <li><a href="">HTML</a></li>
+          <li><a href="">CSS</a></li>
+        </ul>
```
Comment (mckent05) [inline index.html:231]: - Just as the last reviewer mentioned, please ensure that your skills and technology cards are styled precisely as in the Figma template. Please fix this 🙏 

Your project:
- Please add a border-radius to the list of technologies 🙏 
qergqerwg

Design template:
zsas

File: index.html [hunk 4]
```diff
@@ -55,6 +54,228 @@ <h2 class="green">I’m a Full Stack developer from Microverse</h2>
...
+          <h3>
+            Desktop version
+          </h3>
+        </div>
+        <ul class="project-built-with">
+          <li><a href="">Ruby on Rails</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
```
Comment (mckent05) [inline index.html:161]: - Just as the last reviewer mentioned, please ensure that this section is styled precisely as in the Figma template. Please fix this 🙏

Your project:
- Please add a border-radius and ensure they are styled as in the design template 👍 👍 
tetweweg

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @diarisdiakite, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Your portfolio is gradually looking nice, well-done 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (diarisdiakite) [reply index.html:231]: Done @mckent05 @mahbubul14

File: index.html (hunk 4 above)
Comment (diarisdiakite) [reply index.html:161]: Done @mckent05 @mahbubul14

**Comment by diarisdiakite** [comment]
In this milestone, I:
- Increased padding of the technology competences lists elements
- Added border radius to the 2 lists elements
- Changed the PR's name to reflect both sections

File: css/styles.css [hunk 5]
```diff
@@ -4,11 +4,64 @@
...
+  background-color: white;
+}
+
+.see-resume-button:hover {
+  background-color: black;
+}
+
+.see-resume-button:enabled {
+  background-color: white;
 }
```
Comment (AbayomiOlaoye) [inline css/styles.css:65]: - [ ] [OPTIONAL] Great efforts there, @diarisdiakite :clap: however, this `background` value of `black` is not conforming with the requirements on Figma. Kindly check, using the screenshot below as guide to get the correct values onFigma.
image

**Review by AbayomiOlaoye — APPROVED** [review]
Hi @diarisdiakite :clap:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good UI implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 
- Professional ReadMe documentation :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 5 above)
Comment (diarisdiakite) [reply css/styles.css:65]: Well received, that will be corrected @AbayomiOlaoye

**Pull request merged**